enterprise/server/cmd/ci_runner/main.go

```diff
@@ -210,14 +217,19 @@ func RunAllActions(ctx context.Context, cfg *config.BuildBuddyConfig, im *initMe
 		exitCodeName := "OK"
 
 		// Include the repo's download time as part of the first invocation.
-		if !im.reported {
-			im.reported = true
-			ar.log.Printf("Fetched Git repository in %s\n", startTime.Sub(im.start))
-			startTime = im.start
+		if !ws.reportedInitMetrics {
+			ws.reportedInitMetrics = true
+			// Print the setup logs to the first action's logs.
+			b, _ := io.ReadAll(&ws.logBuf)
+			ws.logBuf.Reset()
+			ar.log.Printf("[BUILDBUDDY] Setup completed in %s", startTime.Sub(ws.startTime))
```

Comment:
can we remove the "[BUILDBUDDY] " prefix here and below?

Reply:
Done